dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi12/fat/tests/PackagePrivateAccessTest.java

```diff
@@ -54,8 +56,9 @@ public static void setUp() throws Exception {
                         .addClass("jp.test.bean.MyBean")
                         .add(new FileAsset(new File("test-applications/packagePrivateAccessApp.war/resources/WEB-INF/web.xml")), "/WEB-INF/web.xml");    
         server = LibertyServerFactory.getStartedLibertyServer("packagePrivateAccessServer");
+        server.setMarkToEndOfLog(server.getDefaultLogFile());
         ShrinkHelper.exportDropinAppToServer(server, packagePrivateAccessApp);
-        server.waitForStringInLogUsingMark("CWWKZ0001I.*Application packagePrivateAccessApp started");
+        assertNotNull("packagePrivateAccessApp started or updated message", server.waitForStringInLogUsingMark("CWWKZ000[13]I.*packagePrivateAccessApp"));
```

Comment:
is there anywhere common that could sensibly have a `dropInAndStartApplication` method? rather than repeating this same code?

Reply:
Yes, but I don't like that it changes the mark as a side effect :(

Reply:
You could do a method which would drop in and wait for a start or updated message, but then you would still know that you should set the mark before calling it, so it would reduce the amount of code but hide what some of what was going on and still be easy to screw up.